settings.bash

```diff
@@ -48,3 +48,10 @@ export WORKSPACE_RELEASE_IMAGE=developmentimage/${PROJECT_NAME}:release
 # --privileged
 # -v /dev/input/:/dev/input
 export ADDITIONAL_DOCKER_RUN_ARGS=""
+
+# Make the exec script to talk more for debugging/docker setup purposes.
+# This may also be stated in the command line: $> VERBOSE=true ./exec.bash 
+# export VERBOSE=true
```

Comment:
We should have either VERBOSE or SILENT uncommented right?

Apart from that I'd be okay to merge.

Reply:
No, in nothing is uncommented, it is the default (less output, only the image used and possible re-creations of the container).
There acually is a error message if both are set

Reply:
Okay I see :+1: